start-site/src/test/java/io/spring/start/site/extension/dependency/dgs/DgsCodegenHelpDocumentCustomizerTests.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2012-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.start.site.extension.dependency.dgs;
+
+import io.spring.initializr.generator.io.template.MustacheTemplateRenderer;
+import io.spring.initializr.generator.test.io.TextAssert;
+import io.spring.initializr.generator.test.project.ProjectStructure;
+import io.spring.initializr.web.project.ProjectRequest;
+import io.spring.start.site.extension.AbstractExtensionTests;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Tests for {@link DgsCodegenHelpDocumentCustomizer}.
+ *
+ * @author Brian Clozel
+ */
+class DgsCodegenHelpDocumentCustomizerTests extends AbstractExtensionTests {
+
+	@Autowired
+	private MustacheTemplateRenderer templateRenderer;
+
+	@Test
+	void mavenBuildAddsLinkToMavenCodegenPlugin() {
+
+		assertHelpDocument("maven-project", "dgs-codegen").contains("## GraphQL code generation with DGS")
+			.containsSubsequence("This project has been configured to use the Netflix DGS Codegen plugin.",
```

Comment:
I wonder if that's strictly necessary as any wording change needs the test to be updated. I think we're probably a bit more relaxed for other help document tests. Checking that the proper link according to the build system is specified (and the non matching one is not present) looks the real important bit for me.

Reply:
I've relaxed the tests, thanks!